### PR TITLE
Improve port generation

### LINKED ISSUE #47: Can't start multiple TestxServer at the same time
I would like to use graphql-testx with parallel tests, but if I try to do it they will collead on the same port.

This code simulates my issue:
```js
const server1 = new TestxServer(ITEM_MODEL);
const server2 = new TestxServer(ITEM_MODEL);

// start both servers at the same time
await Promise.all([server1.start(), server2.start()]);
```

### PR DESCRIPTION
**Pull request by vinifarias**
### Description

This PR changes the `portastic` dependency to `get-port` because it reduces the chance of race condition for ports by default.

Resolves #47 #57 

##### Checklist

- [X] `npm test` passes
- [X] `npm run build` works
- [X] tests are included
- [X] documentation is changed or added

### CODE REVIEW

File: src/TestxServer.test.ts [hunk 1]
```diff
@@ -35,33 +35,41 @@ test("test start() and close() methods", async t => {
...
     },
     null,
     "Should throw an error after closing the server (ECONNREFUSED)"
   );
 
   await server.start();
-  const result = await request(httpUrl, queries.findAllItems);
+  const httpUrl2 = await server.httpUrl();
+
+  const result = await request(httpUrl2, queries.findAllItems);
```
Comment (b1zzu) [inline src/TestxServer.test.ts:56]: By using the old httpUrl instant of retrieving a new one we ensure that the url doesn't change after re-start, I think is better to revert this change

**Review by b1zzu — APPROVED** [review]
Awesome work, this will enable the possibility to use multiple servers in parallel :+1:

**Comment by wtrocki** [comment]
I have this issue now when building complex tests cases so really good timing. 
Merging and doing new releases to get that solved! 
Amazing work!

File: src/TestxServer.test.ts (hunk 1 above)
Comment (vinifarias) [reply src/TestxServer.test.ts:56]: I made this change because we can just ensure that when doing a re-start after a stop, but not after a close. The test was working before because the getAvailablePort method was not getting a random port.

Comment (b1zzu) [reply src/TestxServer.test.ts:56]: I see, you are right
